cloud/azure/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "application_name" {
+  type        = string
+  description = "Azure Web App Name"
+}
+
+variable "docker_username" {
+  type        = string
+  description = "Docker username"
+}
+
+variable "docker_repository_name" {
+  type        = string
+  description = "Name of container image"
+}
+
+variable "docker_password" {
```

Comment:
Have you needed this yet?  Our civiform/civiform is public for instance.

Reply:
I've been testing with a demo app in a private repo since the image size is smaller and takes less time to pull. but if it's public, you don't need it.

Reply:
The current thing runs that civiform successfully so would be nice if the default could do that too, or was able with some local config.

Reply:
Suggestion (blocking): if you're going to leave it in for development purposes please add a comment explaining

Reply:
I actually just ended up removing this and decided to test with the public civiform image.